prisma/schema.prisma

```diff
@@ -8,13 +8,20 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+
+enum keyword_status {
+  Inprogress @map("in-progress")
+  Done @map("done")
+}
+
 model search_results {
   id                              Int      @id @default(autoincrement())
-  adswords_count                  Int
+  adswords_count                  Int?     
   keyword                         String
+  keyword_status                  keyword_status @default(Inprogress)
```

Comment:
WHat do you think of naming it the following to be more descriptive:

```suggestion
  processing_status                  keyword_status @default(Inprogress)
```

or:

```suggestion
  scraping_status                  keyword_status @default(Inprogress)
```

Reply:
I think second one is more description